src/crimson/os/seastore/onode.h

```diff
@@ -58,6 +58,9 @@ class Onode : public boost::intrusive_ref_counter<
   virtual const onode_layout_t &get_layout() const = 0;
   virtual onode_layout_t &get_mutable_layout(Transaction &t) = 0;
   virtual ~Onode() = default;
+  laddr_t hint = L_ADDR_MIN;
```

Comment:
I think we can get the hint from `tree_cursor_t::get_key_view()`, which is from `Value::p_cursor`.

So we don't need to store it in the interface class `Onode`.